docs/content/policy-reference.md

```diff
@@ -341,6 +341,7 @@ complex types.
 | Built-in | Description | Wasm Support |
 | ------- |-------------|---------------|
 | <span class="opa-keep-it-together">`value := object.get(object, key, default)`</span> | `value` is the value stored by the `object` at `key`. If no value is found, `default` is returned. | ✅ |
+| <span class="opa-keep-it-together">`value := object.dig(object, path, default)`</span> | `value` is the value stored by the `object` at the nested `path`. If no value is found, `default` is returned. For example: `object.dig({"a": {"b": "c"}}, ["a", "b"], "x")` will result in `"c"`.| ✅ |
```

Comment:
That last "✅" requires another implementation of this built-in, in some C file in `wasm/src/`. It's OK to not do this right away, but we should say "SDK-dependent" then.

Reply:
Ahh whoops, copy-pasta error there! 😳 I've moved it to sdk-dependent for now since I'm on M1 and working out how to make that work locally.